lib/vrl/stdlib/src/encode_logfmt.rs

```diff
@@ -0,0 +1,324 @@
+use std::result::Result;
+use vrl::prelude::*;
+
+mod logfmt {
```

Comment:
I don't think it's necessary to have a separate submodule here.

Reply:
Yeah, I introduced a submodule so that I could write `logfmt::encode` in the implementation of the function. I think it makes it more pleasant to read, but I can remove it if you think that's not necessary.

Reply:
We've typically just had private methods at the top level in these modules. I don't feel strongly either way, but did make @StephenWakely 's suggestion in dbd632c